tools/mtouch/Makefile

```diff
@@ -162,6 +162,7 @@ SIMLAUNCHER_FRAMEWORKS =  \
 	-weak_framework AutomaticAssessmentConfiguration	\
 						\
 	-weak_framework AppClip	\
+	-weak_framework Accessibility	\
```

Comment:
🤔 What is a weak_framework?

Reply:
Is telling the compiler to annotated all symbols of the framework as weak. With weak symbols a strong symbol can step on it. Is like having replazable symbols in a way.https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/WeakLinking.html

Reply:
TL&DR the dynamic linker daemon (dyld) won't crash the process if it can't find a weak framework (useful for older OS running newer executables)